index.js

```diff
@@ -1,9 +1,25 @@
 const Discord = require('discord.js');
-const client = new Discord.Client();
+const client = new Discord.Client(
+    {intents: ["GUILD_MESSAGES", "GUILD_VOICE_STATES", "GUILDS"]}
+);
+
+const fs = require("fs");
 
 const config = require('./config.json');
 const commands = require(`./bin/commands`);
 
+//in case the bot was not configured properly
+if(!config.PREFIX || !config.BOT_TOKEN) {
+    console.error("Error: the configuration file was configured properly.");
+    console.error("Make sure there are no spelling mistakes.");
+    process.exit(1);
+}
+
+//create the recordings directory in case people forget to do so
+try {
+    fs.mkdirSync("./recordings/");
+} catch(ignored) {}
```

Comment:
might want to use `path.join(__dirname, recordings)` in case someone runs the script from a different directory and also catch the errors for `EPERM` or `EACCES`, just in case.